### DISCUSSION TITLE
【2022.11.30】community meeting notes

### DISCUSSION THREAD

**Discussion by Stevent-fei** [sealerio/sealer#1898] [General]
hi all, 今晚8点，有咱们sealer的开发者会议，大家有什么议题可以提前准备哦。

**Comment by YTGhost** [comment]
How to implement image policy for sealer using Kyverno #1899

**Comment by Stevent-fei** [comment]
集群升级模块实现

**Comment by zhy76** [comment]
e2e tests in container

**Comment by kakaZhou719** [comment]
【会议纪要】
1. sealer使用kyverno的实现。kyverno policy放到basefs中，实现的时候需要考虑非sealer集群的app镜像安装，@YTGhost
2. sealer 需要有集群升级的功能，不过先保证build,run的基础功能稳定，可以先有调研设计方案。@Stevent-fei
3. image spec 规范定义预计本周有实现方案。@Starnop
4. e2e代码已经运行成功，PR 已经提交，在对接github action时候启动集群 有点问题（swap off失败），目前继续攻关中。@zhy76，@cubxxw熟悉github action的基础设施配置，可提供帮助。
5. k0s 重构代码已经提交，目前block在对conatinerd runtime的支持上。@starComingup。k3s由@cubxxw 同学继续调研中。
6. 支持load本地容器镜像，方案在调研中。@jsparter
